packages/vite/src/node/server/shortcuts.ts

```diff
@@ -0,0 +1,124 @@
+import colors from 'picocolors'
+import type { ViteDevServer } from '..'
+import { openBrowser } from './openBrowser'
+
+export function bindShortcuts(server: ViteDevServer): void {
+  if (!server.httpServer) return
+
+  const helpInfo =
+    colors.dim('press ') +
+    colors.reset(colors.bold('h')) +
+    colors.dim(' to show help')
+  const quitInfo =
+    colors.dim('press ') +
+    colors.reset(colors.bold('q')) +
+    colors.dim(' to quit')
+
+  server.config.logger.info(
+    colors.dim(`  ${colors.green('➜')}  ${helpInfo}, ${quitInfo}`)
+  )
+
+  let actionRunning = false
+
+  const onInput = async (input: string) => {
+    // ctrl+c or ctrl+d
+    if (input === '\x03' || input === '\x04') {
+      return process.kill(process.pid, 'SIGINT')
+    }
+
+    if (input === 'h') {
```

Comment:
This can be itself a shortcut. This is common for CLI to print the help option in the help itself